pom.xml

```diff
@@ -196,10 +203,39 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.9.0</version>
+                <version>3.8.0</version>
```

Comment:
Question: why did we downgrade?

Reply:
Reverted, I was following the Error Prone guide.